code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/util/NotificationData.kt

```diff
@@ -57,4 +59,6 @@ interface NotificationData {
      * @return the long value for the given key, or null if the key is not found or the value is not a long
      */
     fun getLong(key: String): Long?
+
+    fun getBundleWithAllData(): Bundle
```

Comment:
Being nitpicky here, 
Can we simply this naming to `getBundle()`.... NotificationData.getBundle() still tells its getting the complete data in as a Bundle

Also can we add JavaDocs for this method